src/test/java/teammates/test/cases/action/InstructorEditInstructorFeedbackPageActionTest.java

```diff
@@ -139,6 +140,21 @@ protected InstructorEditInstructorFeedbackPageAction getAction(String... params)
     @Override
     @Test
     protected void testAccessControl() throws Exception {
-        //TODO: implement this
+        dataBundle = getTypicalDataBundle();
+        removeAndRestoreDataBundle(dataBundle);
```

Comment:
Perhaps we can add a reference `typicalDataBundle` to `BaseActionTest` and use that instead? `dataBundle` will then be used only for data specific to the test.